12.0/Dockerfile

```diff
@@ -0,0 +1,106 @@
+FROM ubuntu:bionic
+MAINTAINER Odoo S.A. <info@odoo.com>
+
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
+
+# Generate locale C.UTF-8 for postgres and general locale data
+ENV LANG C.UTF-8
+
+# # Use backports to avoid install some libs with pip
+# RUN echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/backports.list
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN apt-get update \
+        && apt-get install -y --no-install-recommends \
+            ca-certificates \
+            curl \
+            dirmngr \
+            fonts-noto-cjk \
+            gnupg \
+            libssl1.0-dev \
+            node-less \
+            python3-pip \
+            python3-phonenumbers \
+            python3-pyldap \
+            python3-qrcode \
+            python3-renderpm \
+            python3-setuptools \
+            python3-slugify \
+            python3-vobject \
+            python3-watchdog \
+            python3-xlrd \
+            xz-utils \
+        && curl -o wkhtmltox.tar.xz -SL https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz \
```

Comment:
Can we bump this to 0.12.6 (don't forget to update  the checksums - also it should be sha256 now)

Btw, the releases are here now https://github.com/wkhtmltopdf/packaging/releases/tag/0.12.6-1

Reply:
Actually can we just install that from apt? wkhtmltopdf 0.12.5 is in apt